_includes/scripts.html

```diff
@@ -2,6 +2,8 @@
 <script>window.jQuery || document.write('<script src="{{ site.baseurl }}/assets/js/vendor/jquery-slim.min.js"><\/script>')</script>
 
 <script src="{{ site.baseurl }}/assets/js/vendor/popper.min.js"{% if site.github %} integrity="{{ site.cdn.popper_hash }}" crossorigin="anonymous"{% endif %}></script>
+<script src="{{ site.baseurl }}/assets/js/vendor/hammer.min.js"{% if site.github %} integrity="{{ site.cdn.hammer_hash }}" crossorigin="anonymous"{% endif %}></script>
```

Comment:
You need to add it in package.json too for the docs package.

Reply:
It's strange but Popper isn't in our `doc-minify` task 🤔

Reply:
True, indeed. Just leave it as is and I'll give it a go in the next days and let you know. :)